scripts/generate.py

```diff
@@ -0,0 +1,45 @@
+import os
+import json
+
+def generate_json_structure(base_dir, included_folders):
+    structure = {}
+
+    for folder in included_folders:
+        folder_path = os.path.join(base_dir, folder)
+        
+        if os.path.isdir(folder_path):
+            images = []
+            for file in os.listdir(folder_path):
+                if file.endswith('.png') or file.endswith('.jpg'):
+                    images.append(f"{folder}/{file}")
+            
+            # Check for category image existence (folder.png or folder.jpg)
+            print(folder_path)
+            if os.path.exists(os.path.join(base_dir, f"{folder}.png")):
+                category_image = f"{folder}.png"
+            elif os.path.exists(os.path.join(base_dir, f"{folder}.jpg")):
+                category_image = f"{folder}.jpg"
+            elif os.path.exists(os.path.join(base_dir, f"{folder}.jpeg")):
+                category_image = f"{folder}.jpeg"
+            else:
+                raise FileNotFoundError(f"Category image not found for folder '{folder}'")
+
+            structure[folder] = {
+                "category_image": category_image,
+                "images": images
+            }
+
+    return structure
+
+if __name__ == "__main__":
+
+    base_dir = '../osrd-images'
+
+    included_folders = ['railway', 'clock', 'map', 'technical_drawing']
+
+    json_structure = generate_json_structure(base_dir, included_folders)
+
+    with open('image_path.json', 'w') as json_file:
+        json.dump(json_structure, json_file, indent=2)
```

Comment:
We could simply print on stdout the resulted json file.
By doing this, we no longer need `json_file`.

> [!IMPORTANT]
> We should remove all `print` function.

> [!NOTE]
> We can simply use shell redirection in the Dockerfile
> ```dockerfile
> RUN python /app/osrd-images/scripts/generate.py > /app/.../image_path.json
> ``` 

```suggestion
      json.dumps(json_structure, indent=2)
```